modelkit/core/model.py

```diff
@@ -305,6 +305,37 @@ def describe(self, t=None):
                 f"[deep_sky_blue1]load memory[/deep_sky_blue1]: "
                 f"[orange3]{humanize.naturalsize(self._load_memory_increment)}"
             )
+
+        def get_all_dependencies_load_info(my_dict, my_model):
+            for model_name, model_dep in my_model.model_dependencies.models.items():
+                if model_name not in my_dict:
+                    my_dict[model_name] = {
+                        "time": model_dep._load_time,
+                        "memory": model_dep._load_memory_increment,
+                    }
+                    get_all_dependencies_load_info(my_dict, model_dep)
+
+        dependencies_load_info = {}
+        get_all_dependencies_load_info(dependencies_load_info, self)
+        load_time_including_dependencies = (self._load_time or 0) + sum(
+            x["time"] or 0 for x in dependencies_load_info.values()
```

Comment:
1/ maybe set `"time": model_dep._load_time or 0` directly to avoid this
2/ why model_dep._load_time can be 0 ?